### ISSUE TITLE
As a user, I want to download a PDF and cite a published editorial so that I can deposit it or share it in a more recognized academic format.

### ISSUE THREAD

**Issue by rlskoeser** [Princeton-CDH/ppa-django#550]
adapt print styles from startwords; related to #548 

## testing notes
- [x] optional doi field for editorials; should display on editorial page when set 
- [x] optional pdf link for editorials; should display link on editorial page when set 
- [x] print preview should suppress PPA headers and footers, adjust margins and font sizes, hide PDF link if any
- [x] try harvesting from zotero and see what the citation looks like; should include doi when set (author orcid is in the header metadata if we have it, but I'm not sure zotero pulls that)

**Comment by mnaydan** [comment]
Questions:
- Is the DOI supposed to link to the item? The full DOI displayed on editorial page, but took me to doi.org when clicked.
- Link style for PDF functional but could be prettier (incorporates the preceding slash):
![Screenshot 2023-09-01 at 4 02 33 PM](https://github.com/Princeton-CDH/ppa-django/assets/22945600/0ed55484-c1e3-4de7-8934-ead2251123c7)
- Zotero harvested Item Type, Title, Abstract (just beginning of full text), Publication (Princeton Prosody Archive), Date, Language, DOI, ISSN, URL, and Library Catalog. **It did not harvest author.** Author ORCID isn't in there either, and I'm not sure where I would check to find the ORCID. 
Citation looks like: “How to Teach Prosody.” Princeton Prosody Archive, March 6, 2019. https://doi.org/10.5281/zenodo.8292370.

**Comment by mnaydan** [comment]
Also as discussed on Slack I think the Publication should be "Princeton Prosody Archive Editorial"

**Comment by rlskoeser** [comment]
@mnaydan the doi won't resolve if it hasn't been published yet; if you want you could test with a SW doi that has been published.

Ugh about author metadata for zotero! That's important, I'll investigate.

I thought I fixed that underline on the slash next to the pdf, I'll look again.

I can update journal title to add "Editoiral" as we discussed. Seem ok to you to still use the main ISSN for PPA?

**Comment by rlskoeser** [comment]
I confirmed that the author orcid is in the metadata in the header.

**Comment by mnaydan** [comment]
I did test with a published DOI (CDH Charter) and it still went to doi.org

**Comment by rlskoeser** [comment]
@mnaydan thanks for testing the doi! I'll look into it

**Comment by rlskoeser** [comment]
@mnaydan I've updated the code and pushed changes to the test site that should address the DOI url problem and also fix the metadata citation problem you found; please confirm when you're able or let me know if there are still problems.

**Comment by mnaydan** [comment]
@rlskoeser 
1. Tested with an existing CDH charter DOI & it linked as expected. Fixed! 
2. PDF underline style fixed.
3. Zotero harvested author and editor! Looks good :) 

Also for documentation's sake, we changed our mind on Slack and the journal title will stay "Princeton Prosody Archive," as we decided there wasn't really any benefit to distinguishing between PPA & PPA Editorial & taking S&Co as a model.
